src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -363,18 +363,18 @@ private async Task<SearchResult> SearchImpl(SqlSearchOptions sqlSearchOptions, S
                             out Stream rawResourceStream);
                         numberOfColumnsRead = reader.FieldCount;
 
-                        // If we get to this point, we know there are more results so we need a continuation token
-                        // Additionally, this resource shouldn't be included in the results
-                        if (matchCount >= clonedSearchOptions.MaxItemCount && isMatch)
+                        string rawResource;
+                        await using (rawResourceStream)
```

Comment:
Why the change to await using?

Reply:
Stream now implements IAsyncDisposable, so we can use this:
![image](https://user-images.githubusercontent.com/197221/197606144-dca3cce4-9112-4e65-8657-863c9dd99e17.png)